### ISSUE TITLE
[$250] Scan - "Only visible to you" whisper is not visible on multiple request report

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#42259]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** v1.4.74-1
**Reproducible in staging?:** Y
**Reproducible in production?:** N
**If this was caught during regression testing, add the test name, ID and link from TestRail:** https://expensify.testrail.io/index.php?/tests/view/4561920
**Email or phone of affected tester (no customers):** dave0123seife+employe16a@gmail.com
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Issue reported by:** Applause-Internal team

## Action Performed:
1. Navigate to the workspace chat as the employee
2. Submit a request using a scan
3. While it is Scanning, There's a whisper message on top of the request "Only visible to you" (this might take a few sec to show up)
4. Submit a manual request.
5. Because we have not submitted the requests, Notice both requests are in one request conversation,
6. Notice that there is no whisper message on top of the request "Only visible to you"

## Expected Result:
"Only visible to you" whisper is visible on multiple request report

## Actual Result:
"Only visible to you" whisper is not visible on multiple request report

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [x] Android: mWeb Chrome
- [ ] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/Expensify/App/assets/115492554/c64d536b-f7d5-4842-a698-96b130c3b91b

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01ba739091862c309e</li>
        <li>Upwork Job ID: 1791487887666659328</li>
        <li>Last Price Increase: 2024-05-17</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @hungvu193</details>

**Comment by izarutskaya** [comment]
@CortneyOfstad I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors.

**Comment by izarutskaya** [comment]
We think this issue might be related to the #collect project.

**Comment by Beamanator** [comment]
haven't been able to look at this, site has been down recently and fixed other blockers earlier 😅

**Comment by Beamanator** [comment]
I'm pretty sure this can be worked externally - Also we won't deploy today so we have a bit more time to get external help

**Comment by CortneyOfstad** [comment]
Sounds good @Beamanator — does this still need to remain hourly or can I adjust this to daily?

**Comment by hungvu193** [comment]
So I think can close it then?

**Comment by CortneyOfstad** [comment]
@Beamanator can you confirm if this is a bug or not based on the comment [here](https://github.com/Expensify/App/issues/42259#issuecomment-2118134056)?

**Comment by Beamanator** [comment]
Oofda quite interesting!! @ishpaul777 @yuwenmemon In this issue OP's video, it looks like we show the "Only visible to you" when smart scanning a receipt, but when we do a manual request & the expense previews are combined, we do NOT show "Only visible to you" - is that intentional? 

From @ishpaul777 's comment [here](https://github.com/Expensify/App/issues/42259#issuecomment-2118134056), it sounds like we're trying to remove the "Only visible to you" message... Maybe everywhere?

**Comment by hungvu193** [comment]
Cool. I also think this is intentional

**Comment by Beamanator** [comment]
For now I'll at least make this NAB 👍

**Comment by Beamanator** [comment]
aah ya sorry that's what i meant @ishpaul777 - i meant "it looks like we're trying to remove the "Only visible to you" message for ALL smartscan previews" -> The reason I'm mentioning that, is that this issue is mainly about "multiple-request report previews" but I think it could be generalized

**Comment by Beamanator** [comment]
Yeah exactly, that makes sense to me too - so IFF we should be getting rid of that, I think we should make a new issue OR at the least bit, completely clean up this issue OP & title 😅

**Comment by Beamanator** [comment]
cc @yuwenmemon , maybe even @Expensify/design - can y'all confirm if ^ is a bug?

**Comment by shawnborton** [comment]
@grgia can you confirm expected behavior here? My understanding is with the updated pending transactions project, we should never see the "only you can see this" whisper pattern with scanning expenses.

**Comment by grgia** [comment]
Yes this is expected as of right now, since we're removing whispers from scan. We're still waiting on [this BE change](https://github.com/Expensify/Expensify/issues/385496) to remove the whisper logic entirely, but we stopped showing it in the FE

**Comment by Beamanator** [comment]
Nice, thanks - yeah since it looks like we're missing one place, @grgia do you think we should work on that in this issue OR create a fresh one for that 1 spot? OR do you think the backend PRs should fix this? 🙏

**Comment by hungvu193** [comment]
little bump @grgia

**Comment by grgia** [comment]
@Beamanator the BE PR is removing whispers from pending/scanning receipts

**Comment by hungvu193** [comment]
@Beamanator I just want to clarify if this issue is treated as a regression, if yes please unassign me 😄

**Comment by Beamanator** [comment]
@grgia sorry but it's still not quite clear to me if the BE PR will fix the "bug" shown here - https://github.com/Expensify/App/issues/42259#issuecomment-2122495670 - that there is still 1 final place we DO show "Only visible to you" where we shouldn't.

If you can confirm that the BE PR will fix that, I think we're good to close this one out, yeah? 🙏

**Comment by hungvu193** [comment]
@grgia  can you please confirm above [comment](https://github.com/Expensify/App/issues/42259#issuecomment-2132842210)?

**Comment by grgia** [comment]
@Beamanator we're removing whispers from all newly created pending/scanning transactions, so we shouldn't see "only visible to you" at all. We could remove it from the UI as part of this issue for existing transactions though. Does that answer your question?

**Comment by Beamanator** [comment]
Ahhhh the point is that the whisper already exists for old transactions, but shouldn't happen for new ones, eh?

> We could remove it from the UI as part of this issue for existing transactions though. 

Has this been discussed anywhere in the doc or other threads @grgia ? If not, maybe we should bring it up somewhere?

**Comment by grgia** [comment]
Yeah we can bring this up in slack, it honestly won't affect many reports since they are only pending or scanning for a day or two. I take it back on fixing it in this issue, I'm pretty sure we can close this one in favor of the BE changes @Beamanator. Happy to jump into a discussion in slack if there's still something that's uncertain here

**Comment by Beamanator** [comment]
Ok cool I'm also down with closing in favor of BE changes, let's go for it 👍
